python/README.md

```diff
@@ -26,57 +36,26 @@ optional arguments:
   -port PORT, --flightport PORT
                         Dremio flight server port. Defaults to 32010.
   -user USERNAME, --username USERNAME
-                        Dremio username. Defaults to "dremio".
+                        Dremio username.
   -pass PASSWORD, --password PASSWORD
-                        Dremio password. Defaults to "dremio123".
-  -pat PAT_OR_AUTH_TOKEN, --personalAccessToken PAT_OR_AUTH_TOKEN, -authToken PAT_OR_AUTH_TOKEN, --authToken PAT_OR_AUTH_TOKEN
+                        Dremio password.
+  -pat TOKEN, --token TOKEN
                         Either a Personal Access Token or an OAuth2 Token.
   -query QUERY, --sqlQuery QUERY
-                        SQL query to test.
+                        SQL query to test
   -tls, --tls           Enable encrypted connection. Defaults to False.
-  -dsv DISABLE_SERVER_VERIFICATION, --disableServerVerification DISABLE_SERVER_VERIFICATION
-                        Disable TLS server verification. Defaults to False.
-  -certs TRUSTED_CERTIFICATES, --trustedCertificates TRUSTED_CERTIFICATES
+  -dcv DISABLE_CERTIFICATE_VERIFICATION, --disableCertificateVerification DISABLE_CERTIFICATE_VERIFICATION
+                        Disables TLS server verification. Defaults to False.
+  -path_to_certs PATH_TO_CERTS, --trustedCertificates PATH_TO_CERTS
                         Path to trusted certificates for encrypted connection. Defaults to system certificates.
-  -sessionProperties [SESSION_PROPERTIES ...], --sessionProperties [SESSION_PROPERTIES ...]
-                        Key value pairs of SessionProperty, example: -sessionProperties schema='Samples."samples.dremio.com"'
+  -sp [SESSION_PROPERTIES ...], --sessionProperty [SESSION_PROPERTIES ...]
+                        Key value pairs of SessionProperty, example: -sp schema='Samples."samples.dremio.com"' -sp key=value
   -engine ENGINE, --engine ENGINE
```

Comment:
Have we tested this? Also this is Cloud only.

Reply:
Have not tested. Will test now and document it being cloud only.